ui/hooks/accounts/useMultichainClient.ts

```diff
@@ -7,26 +7,46 @@ import {
   multichainUpdateBalance,
 } from '../../store/actions';
 import { BITCOIN_WALLET_SNAP_ID } from '../../../shared/lib/accounts/bitcoin-wallet-snap';
+import { SOLANA_WALLET_SNAP_ID } from '../../../shared/lib/accounts/solana-wallet-snap';
+
+export enum WalletClientType {
+  Bitcoin = 'bitcoin-wallet-snap',
+  Solana = 'solana-wallet-snap',
+}
+const SNAP_ID_MAP: Record<WalletClientType, string> = {
+  [WalletClientType.Bitcoin]: BITCOIN_WALLET_SNAP_ID,
+  [WalletClientType.Solana]: SOLANA_WALLET_SNAP_ID,
+};
+
+export class MultichainSender implements Sender {
```

Comment:
I would also rename this to `MultichainWalletSnapSender` just so we know it's related to our "wallet snaps", WDYT?

Reply:
Yap, sounds good 👍🏼